lab5/solid_principle.py

```diff
@@ -0,0 +1,149 @@
+from math import pi
+
+# --------------------------------------------------------------------------------------------
+# SRP - Single Responsibility Principle
+class Customer:
+    """
+    Create a customer.
+    """
+    def __init__(self, name, mobile):
+        self.name = name
+        self.mobile = mobile
+
+    def get_name(self):
+        return self.name
+
+    def get_mobile(self):
+        return self.mobile
+
+class Purchase(Customer):
```

Comment:
in this case, inheritance is not a great option, it's better to use aggregation here